pkg/remote/aws/repository/sns_repository.go

```diff
@@ -0,0 +1,34 @@
+package repository
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sns"
+	"github.com/aws/aws-sdk-go/service/sns/snsiface"
+)
+
+type SNSRepository interface {
+	ListAllTopics() ([]*sns.Topic, error)
+}
+
+type snsRepositoryImpl struct {
```

Comment:
Is the `Impl` suffix required there ?

Reply:
No you're right it survived a refacto :joy: